docker/ubuntu/kodeb/Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG REGISTRY=docker.io
+FROM $REGISTRY/koreader/kobase:0.3.2-20.04 AS base
+
+FROM base AS build
+
+USER root
+RUN apt-get update
+RUN apt-get upgrade -y
+ARG TOOLS="chrpath dpkg-deb xz"
```

Comment:
`dpkg-deb` and `xz` are already in the base image but chrpath is never installed here?